docs/index.md

```diff
@@ -9,12 +9,14 @@ hide:
   ![Kolena](assets/images/wordmark-white.svg#only-dark){ width="400" }
 </figure>
 
+<!-- markdownlint-disable MD013 -->
```

Comment:
It's not great to have to disable this all over the place — do we ever need `MD013`?

Reply:
[MD013](https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md) is the rule for line length